polkadot/runtime/parachains/src/scheduler.rs

```diff
@@ -584,7 +584,8 @@ impl<T: Config> Pallet<T> {
 			return
 		}
 		// If there exists a core, ensure we schedule at least one job onto it.
-		let n_lookahead = Self::claimqueue_lookahead().max(1);
+		// TODO (now): Add a test for this. This wax `max` and nobody noticed.
+		let n_lookahead = Self::claimqueue_lookahead().min(1);
```

Comment:
This was max out of the reason to always have at least 1 lookahead.

Reply:
Because `0` makes no sense.

Reply:
LOL. Just realizing I was after all holding it wrong. Thanks.